### ISSUE TITLE
Design individual domain error page 

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#395]
Each domain should have it's own a11y error page, based off of the style of the [about page](https://pulse.cio.gov/about/), with the errors grouped by category and each category being expandable and having an anchor tag so that we can link directly to the section from each domain's expanded information section on the [domains page](https://staging.pulse.cio.gov/accessibility/domains/).  

For each error, the error type, message, context, and selector should be included.

**Comment by gbinal** [comment]
![screen shot 2016-03-22 at 11 40 43 am](https://cloud.githubusercontent.com/assets/633088/13957486/09af93ae-f023-11e5-8e97-ef22283547fa.png)
![screen shot 2016-03-22 at 11 38 23 am](https://cloud.githubusercontent.com/assets/633088/13957485/09ac5d06-f023-11e5-91af-8303b86cbd66.png)
![screen shot 2016-03-22 at 11 38 10 am](https://cloud.githubusercontent.com/assets/633088/13957487/09b6d0f6-f023-11e5-8c86-3ffacee411ec.png)
